lib/chat/user_message_rows.dart

```diff
@@ -129,7 +129,7 @@ buildMessageRows(context) {
                         )
                       ),
                       children: [
-                        if(dummyData[i]["isSent"]) WidgetSpan(
+                        if(dummyData[i]["isSent"] != null) WidgetSpan(
```

Comment:
does this still work for when `dummyData[i]["isSent"]` is `false`? Does that mean `false == null`? or should this be `if(dummyData[i]["isSent"] != false`?

Reply:
Oh, good catch! I definitely wasn't paying attention there -- thanks! I'll fix after work. 👍